src/python/WMCore/WebTools/Root.py

```diff
@@ -269,6 +269,7 @@ def _configureCherryPy(self):
                                         'tools.secmodv2.group': self.secconfig.default.group,
                                         'tools.secmodv2.site': self.secconfig.default.site})
         cherrypy.config.update({'tools.cpstats.on': configDict.get('cpstats', False)})
+        cherrypy.config.update({'server.statistics': configDict.get('cpstats', False)})
```

Comment:
Do we want to always, either enable both or don't enable any at all? Otherwise we should read it from a different service parameter.